ext_mongodb.php

```diff
@@ -354,23 +357,34 @@ public function __construct(mixed $filter, array $options = array())
 			$this->query['flags'] = array_key_exists('flags', $options ) ? (int) $options['flags'] : 0;
 			$this->query['limit'] = array_key_exists('limit', $options ) ? (int) $options['limit'] : 0;
 			$this->query['skip'] = array_key_exists('skip', $options ) ? (int) $options['skip'] : 0;
-		}
 
-		if (array_key_exists('modifiers', $options)) {
-			Utils::mustBeArrayOrObject('modifiers', $options['modifiers']);
-			foreach ($options['modifiers'] as $key => $value) {
-				$this->query['query'][$key] = $value;
+			if (array_key_exists('readConcern', $options)) {
+				if (!($options['readConcern'] instanceof \MongoDB\Driver\readConcern)) {
+					throw new \MongoDB\Driver\Exception\InvalidArgumentException(
+						'Expected "readConcern" option to be MongoDB\Driver\ReadConcern, ' .
+						gettype($options['readConcern']) . ' given'
+					);
+				} else {
+					$this->query['readConcern'] = $options['readConcern']->getLevel();
```

Comment:
Note: if PHPC ends up printing the read concern, it will recursively invoke the ReadConcern class' debug handler for the field. In that case, I think this output would need to mimic the same. Would it be best to store the ReadConcern instance internally (cloned if we need to keep it immutable)?